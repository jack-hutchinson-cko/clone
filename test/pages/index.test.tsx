import renderer from 'react-test-renderer'
import IndexPage from '../../pages/index'

describe('Index page', () => {
  it('should match the snapshot', () => {
    const posts = [
      {
        userId: 1,
        id: 1,
        title: 'title',
        body: 'body',
      },
    ]
    const tree = renderer.create(<IndexPage posts={posts} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
