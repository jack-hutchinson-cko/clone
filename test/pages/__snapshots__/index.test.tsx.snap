// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<main>
  <h1>
    Checkout documentation
  </h1>
  <div
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <a>
      title
    </a>
  </div>
  <p>
    Will come soon...
  </p>
</main>
`;
